metadata/lang/fr.yml

```diff
@@ -24,9 +24,31 @@ supporting-galaxies: "Disponible sur ces instances de Galaxy"
 last-modification: "Dernière modification"
 
 
-# other boxes
+# boxes
 tip: "Conseil"
-handson: "En pratique"
+hands-on: "En pratique"
+agenda: "Agenda"
+code-in: 'Entrée'
+code-out: 'Sortie'
+comment: 'Commentaire'
+details: 'Détails'
+solution: 'Solution'
+warning: 'Attention'
+question: 'Question'
+
+# contributions
+authors: "Author(s)"
```

Comment:
@yvanlebras could you add some french translations for these contribution type terms?

Reply:
or @bebatut perhaps?

Reply:
@Marie59 ... :)

Reply:
Is there Galaxy community agreement on how to handle non-gendered to gendered word translation (gender-neutral / gender-inclusive language)?

Was going to contribute some French but going English -> French is [not straightforward on a word-by-word basis and there are a few ways we could go about](https://medium.com/language-lab/how-to-use-gender-neutral-language-in-german-and-french-31d72712b0c0).

Reply:
Hello !
Indeed, french is a bit tricky, for instance author in french it's auteur for a man and woman it can be both auteure or autrice 😅 I can write it as I feel but someone need to check it out afterwards

Reply:
For the contribution types, for example "editors", in English we could also use the word for the role/activity, so "Editing", in case that is useful for the translations. These words will be used at the top of a tutorial to indicate the people involved in each of the tasks. e.g. "Editing: Marie, Saskia" or "Editors: Marie, Saskia" so both of those would be fine for the translations as well.